File: solutions/215.Kth_Largest_Element_in_an_Array/AC_sort_nlogn.cpp
/*
*  Author:      illuz <iilluzen[at]gmail.com>
*  File:        AC_sort_nlogn.cpp
*  Create Date: 2015-08-11 20:07:33
*  Descripton:   
*/

#include <bits/stdc++.h>

using namespace std;
const int N = 0;

class Solution {
public:
    int findKthLargest(vector<int>& nums, int k) {
        sort(nums.begin(), nums.end());
        return nums[nums.size() - k - 1];
    }
};

int main() {

    return 0;
}

File: solutions/215.Kth_Largest_Element_in_an_Array/AC_sort_nlogn.py
#!/usr/local/bin/python
# -*- coding: utf-8 -*-
# Author:      illuz <iilluzen[at]gmail.com>
# File:        AC_sort_nlogn.py
# Create Date: 2015-08-11 20:41:43
# Usage:       AC_sort_nlogn.py 
# Descripton:  


class Solution:
    # @param {integer[]} nums
    # @param {integer} k
    # @return {integer}
    def findKthLargest(self, nums, k):
        return sorted(nums)[-k]
        